webpack/webpack.common.js

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2017. The Hyve and respective contributors
```

Comment:
Please relicence to sth like:
/**
 * Copyright (C) 2019  Stichting PALGA
 * This file is distributed under the GNU Affero General Public License
 * (see accompanying file LICENSE).
 */